odbc/src/handles.rs

```diff
@@ -143,6 +186,19 @@ impl Statement {
             connection,
             _attributes: Box::new(StatementAttributes::default()),
             state,
+            errors: vec![],
         }
     }
 }
+
+#[derive(Debug)]
+pub struct Descriptor {
+    pub errors: Vec<ODBCError>,
+}
+
+impl Descriptor {
+    #[allow(dead_code)]
+    pub fn default() -> Descriptor {
+        Self { errors: vec![] }
+    }
+}
```

Comment:
```suggestion
```
We should derive the `Default` trait here instead of implementing our own `default()` func

Reply:
Done